src/kindlemint/agents/kdp_performance_agent.py

```
@@ -0,0 +1,652 @@
+                        # Create monitoring task
+                        task = Task(
+                            task_type="monitor_book",
+                            task_data={
+                                "type": "monitor_book",
+                                "book_id": book_id,
+                                "asin": book_data["asin"],
+                            },
+                            required_capabilities=[
+                                AgentCapability.PERFORMANCE_MONITORING
+                            ],
+                        )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'task_data' in constructor call

(E1123)